shopBuddy/shopBuddy/ShoppingList+Persistent.m

```diff
@@ -0,0 +1,53 @@
+//
+//  ShoppingList+Persistent.m
+//  shopBuddy
+//
+//  Created by Rachna Gupta on 7/21/22.
+//
+
+#import "ShoppingList+Persistent.h"
+#import "Item.h"
+#import "ShoppingList.h"
+
+@implementation ShoppingList (Persistent)
+
+- (void) saveList: (ShoppingList *)list {
+    [self saveInBackground];
+}
++ (void) createEmptyList:(NSString *)store_name withCompletion: (PFBooleanResultBlock  _Nullable)completion {
```

Comment:
completion block should return `ShoppingList *` otherwise how can caller get access to the new list instance?